### DISCUSSION TITLE
Tenants database column requires unique values

### DISCUSSION THREAD

**Discussion by cannycookie** [spatie/laravel-multitenancy#330] [Q&A]
This does not allow for using a single database for multi tenants.

**Comment by masterix21** [comment]
why?

**Reply by cannycookie** [reply]
the default database column with a unique index requires every site to have it's own database.  I've just dropped the column

**Comment by chrisan** [comment]
@masterix21 is dropping the database column the suggested action for a single database deployment?

`$table->string('database')->unique();` is included as part of the `create_landlord_tenants_table` migration

**Reply by masterix21** [reply]
Make it nullable
